transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,443 @@
+import random
+from initialize import spacy_nlp
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+# defaults
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+
+# intialize
+try:
+    conjugate_en(verb='testing',tense='present',number='singular')
+except:
+    pass
+
+
+class NegateStrengthen(SentenceAndTargetOperation):
```

Comment:
Also, it would be great if you could add few lines of comment for this transformation for better readability.

Reply:
Can we add few lines of comment here, please?

Reply:
Ok done!